icons/yt.svg

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
These SVG files should be cleaned up with SVGO. I’ll resize them all for you and will share them :)

Reply:
Fixed